src/main/java/io/jenkins/plugins/synopsys/security/scan/service/ScannerArgumentService.java

```diff
@@ -181,19 +181,46 @@ private void setScanObject(BridgeInput bridgeInput, Object scanObject, Object sc
             setCoverityProjectNameAndStreamName(coverity, scmObject);
             bridgeInput.setCoverity(coverity);
         } else if (scanObject instanceof Polaris) {
+            Polaris polaris = (Polaris) scanObject;
+            setPolarisProjectNameAndApplicationName(polaris, scmObject);
             bridgeInput.setPolaris((Polaris) scanObject);
         }
     }
 
     private void setCoverityProjectNameAndStreamName(Coverity coverity, Object scmObject) {
-        String repositoryName = getRepositoryName(scmObject);
+        String gitURL = envVars.get(ApplicationConstants.ENV_GIT_URL_KEY);
+        String repositoryName = gitURL.replaceFirst("^.*\\/(.+?)\\.git$", "$1");
```

Comment:
Need to check, should we extract repository name from git url ?

Can we use `currentBuild.projectName` from Global Variables Reference? As I can see, GitLab and ADO are using BUILD information in Integrations Default Values confluence page.

In the confluence page, GitHub, GitLab and ADO default values are specified, so we should decide first what values we are going to use for Jenkins.

cc: @jahid1209